main.js

```diff
@@ -1 +1,69 @@
+const emojis = document.querySelectorAll('.emoji')
+const visibleEmojis = document.querySelector('.emojis__slider')
+const slider = document.querySelector('.slider__content')
+const prevBtn = document.querySelector('.prev')
+const nextBtn = document.querySelector('.next')
+const result = document.querySelector('.result')
+let emojisMap = new Map()
 
+const moveRight = () => {
+    if (slider.classList.contains('slide-to-center')) {
+        slider.classList.remove('slide-to-left')
+        slider.classList.remove('slide-to-center')
+    }
+    if (slider.classList.contains('slide-to-left')) {
+        slider.classList.remove('slide-to-left')
+        slider.classList.add('slide-to-center')
+    }
+    if (slider.classList.contains('slide-to-right')) {
+        slider.classList.remove('slide-to-right')
+        slider.classList.remove('slide-to-center')
+    }
+    slider.classList.remove('slide-to-left')
+    slider.classList.add('slide-to-right')
+}
+
+const moveLeft = () => {
+    if (slider.classList.contains('slide-to-center')) {
+        slider.classList.remove('slide-to-right')
+        slider.classList.remove('slide-to-center')
+    }
+    if (slider.classList.contains('slide-to-right')) {
+        slider.classList.remove('slide-to-right')
+        slider.classList.add('slide-to-center')
+    }
+    if (slider.classList.contains('slide-to-left')) {
+        slider.classList.remove('slide-to-left')
+        slider.classList.remove('slide-to-center')
+    }
+    slider.classList.remove('slide-to-right')
+    slider.classList.add('slide-to-left')
+}
+
+const render = () => {
+    const emojiKeys = Array.from(emojisMap.values())
+    result.replaceChildren(...emojiKeys)
```

Comment:
aa

Reply:
+ you can simply pass `emojiKeys` without spread

Reply:
this is what shows without spread instead of emoji: [object HTMLParagraphElement]